lib/urlwatch/worker.py

```diff
@@ -53,10 +53,10 @@ def run_parallel(func, items):
 def run_jobs(urlwatcher):
     cache_storage = urlwatcher.cache_storage
     jobs = [job.with_defaults(urlwatcher.config_storage.config)
-            for job in urlwatcher.jobs]
+            for (idx, job) in enumerate(urlwatcher.jobs) if ((idx + 1) in urlwatcher.urlwatch_config.joblist or (len(urlwatcher.urlwatch_config.joblist) == 0))]
```

Comment:
Might be good before filtering the list of jobs to filter out:

    if not all(1 <= idx <= len(urlwatcher.jobs) for idx in urlwatcher.urlwatch_config.joblist):
        raise ValueError(f'All job indices must be between 1 and {len(urlwatcher.jobs)}: {urlwatcher.urlwatch_config.joblist}')

Reply:
Also `len(urlwatcher.urlwatch_config.joblist) == 0` should be the same as `not urlwatcher.urlwatch_config.joblist`

Reply:
Good ideas. Should I create a new commit with these suggestions?